starter/source/materials/mat/mat051/nrf51ini.F

```diff
@@ -72,7 +72,7 @@ SUBROUTINE NRF51INI(IPM             ,     PM,      X, NIX,    IX,
 C-----------------------------------------------
 C   D u m m y   A r g u m e n t s
 C-----------------------------------------------
-      INTEGER,INTENT(IN)    :: NIX,IX(NIX,NEL),IPM(NPROPMI,NUMMAT),NV46,NUVAR,NEL
+      INTEGER,INTENT(IN)    :: NIX,IX(NIX,*),IPM(NPROPMI,NUMMAT),NV46,NUVAR,NEL
```

Comment:
Hi Thierry , apparently you removed the dimension here , can you correct ?

Thanks,

Reply:
Sorry I didn't read you comment

Reply:
I reset what was already defined before my previous change yesterday. It was not correct to replace '*' by 'NEL' which is group size and not total element number.
I could also add an additional argument to bring expected size in the subroutine.

Reply:
I updated nrf51init.F : NUMEL is new argument (total number of elements) which is used to define IX size :  IX(NIX,NUMEL)